docs/cli.md

```diff
@@ -189,25 +203,146 @@ BigFlow can build your workflows to Airflow DAGs and deploy them to Google Cloud
 
 There are two build artifacts:
 
-1. DAG files with workflow definitions,
-1. Docker image with workflows code.
+1. Airflow DAG files with workflows definitions,
+1. Docker image with workflows computation code.
 
 There are four `build` commands:
 
-build-dags,build-image,build-package,build
-
-
-1. `build-dags` generates Airflow DAG files from all your workflows, one file for each workflow.
+1. `build-dags` generates Airflow DAG files from your workflows. 
+    DAG files are saved to the local `.dags` dir.
 1. `build-package` generates a PIP package from your project based on `project_setup.py`.
 1. `build-image` generates a Docker image with this package and all requirements.
 1. `build` simply runs `build-dags`, `build-package`, and `build-image`.
 
-// TODO 
+Start from getting detailed help: 
+
+```bash
+bigflow build-dags -h
+bigflow build-package -h
+bigflow build-image -h
+bigflow build -h
+```
+
+Before using `build` commands make sure that you have
+a valid `deployment_config.py` file.
+It should define the `docker_repository` parameter. 
+Read more about `deployment_config.py` in
+[Managing configuration in deployment_config.py](#managing-configuration-in-deployment_configpy).
+
+
+### Building DAG files
+
+The `build-dags` command takes two optional parameters:
+
+* `--start-time` &mdash; the first [runtime](workflow-and-job.md#the-runtime-parameter)
+  of your workflows. If empty, a `current hour` is used for hourly workflows and 
```

Comment:
Why  `current hour` and `today` are written as code?

Reply:
it's not now, minutes and seconds are truncated

Reply:
`--start-time` &mdash; the first [runtime](workflow-and-job.md#the-runtime-parameter)
  of your workflows. If empty, a current hour (`datetime.datetime.now().replace(minute=0, second=0, microsecond=0)`)
  is used for hourly workflows and `datetime.date.today()` for daily workflows.